blog/2021-06-17-activity-timeouts.md

```diff
@@ -0,0 +1,258 @@
+---
+tags:
+  - Temporal
+  - architecture
+  - timeouts
+posted_on_: 2021-06-17T00:00:09Z
+slug: activity-timeouts
+title: 'The 4 Types of Activity Timeouts'
```

Comment:
@flossypurse: Title style in docs is sentence case. Blog-post titles are currently a mix of sentence case and title case. Going forward, should we use sentence case for the blog as well as the docs?

Side note: Usually, style calls for spelling out numbers zero through nine. But for blog-post title, I'm inclined to stet.

Reply:
```suggestion
title: 'The 4 Types of Activity timeouts'
```